python/Ganga/new_tests/TestProxy.py

```diff
@@ -79,7 +85,7 @@ def test_dict_attributes(self):
         self.assertFalse(hasattr(proxied, '_impl'))
 
         # A proxy instance should only have _impl in its dict
-        self.assertEqual(self.p.__dict__.keys(), ['_impl'])
+        self.assertEqual(self.p.__dict__.keys(), ['_impl', '_proxyObject'])
```

Comment:
I'm NOT talking about the Proxy implementation here.

I'm concerned that if line 73 reads assertTrue then I get I get a proxyObject attribute ADDED by the test. It doesn't matter that 73 then fails.
The failing test is somehow adding a ```_proxyObject``` attribute to the object being tested here at line 88.

If line 73 reads assertFalse then this test should NOT have a ```_proxyObject``` and the conversation is mute, I'm concerned about the test behaviour here, not ganga.